pallets/ocw-oracle/Cargo.toml

```diff
@@ -23,6 +23,8 @@ sp-std        = { git = 'https://github.com/paritytech/substrate.git', branch =
 default-features = false
 version          = '0.2'
 # features = ['libm']    # <--- Uncomment if you wish to use `Float` and `Real` without `std`
```

Comment:
We don't need this lib, with blockchain it must be determinism, float is not meet this requirement. We need also remove other comments or non-used code.

Reply:
got it, this lib will only be used in OCW, convert f64 to u64,
maybe we need it here, but I'll check it in more detail.